examples/golang/put-bucket-notification-creation/put-bucket-notification-creation.go

```diff
@@ -0,0 +1,90 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/credentials"
+	"github.com/aws/aws-sdk-go/aws/endpoints"
+	"github.com/aws/aws-sdk-go/aws/session"
+	"github.com/aws/aws-sdk-go/service/s3"
+)
+
+func main() {
+	bucket := flag.String("b", "", "Name of the bucket to add notification to")
+	topic := flag.String("t", "", "The topic onto which the notification is attached to")
+	flag.Parse()
+
+	if *bucket == "" {
+		fmt.Println("You must supply the name of the bucket")
+		fmt.Println("-b BUCKET")
+		return
+	}
+
+	if *topic == "" {
+		fmt.Println("You must supply the name of the topic")
+		fmt.Println("-t TOPIC")
+		return
+	}
+
+	//Ceph RGW Credentials
+	access_key := "0555b35654ad1656d804"
+	secret_key := "h7GhxuBLTrlhVUyxSPUKUV8r/2EI4ngqJxD7iBdBYLhwluN30JaT3Q=="
+	token_id := ""
```

Comment:
what is the "token_id" used fro?

Reply:
It is a mandatory argument.
It is used to prove that the user has been authenticated.
Here is a link for it https://stackoverflow.com/questions/48543948/aws-cognito-whats-the-difference-between-access-and-identity-tokens